Makefile

```diff
@@ -1,11 +1,12 @@
-export VERSION := 0.1.0
+export VERSION := 0.2.0
 OWNER ?= elastic
 REPO ?= go-licenser
 TEST_UNIT_FLAGS ?= -timeout 10s -p 4 -race -cover
 TEST_UNIT_PACKAGE ?= ./...
-LINT_FOLDERS ?= $(shell go list ./... | sed 's|github.com/$(OWNER)/$(REPO)/||' | grep -v github.com/$(OWNER)/$(REPO))
```

Comment:
Out of curiosity, why did we remove this?

Reply:
Because it's not required for this project as it doesn't have vendors. It was useful for others that did have vendors 🙂